main.cpp

```diff
@@ -12,76 +12,202 @@ What you create in this project will be used as the basis of Project 5 in the co
+        Arrays have not been discussed and can't be used in this project.
+        Instead, we can use an Integer primitive to store the Number of Pixels:
+
+    Display:
+        Number of Pixels
+        Amount of Power consumed (milliwatt-hours)
+        Brightness
+        width in cm
+        height in cm
+************************
+
+1) Fill in #5 - #9 with plain-english UDTs for the 5 properties you created for UDT #10
+    example: 
+        If #10's first property was 'Engine', then your `Thing 5)` will be `Engine` and 
+        you will need to provide 5 properties and 3 member functions of that Engine object in plain English
 */
+
+/*
+Thing 1) Restaurant
+5 properties:
+    1) Number of tables
+    2) Number of employees
+    3) Restaurant name
+    4) Size of restaurant in square feet
+    5) Amount of profit each day
+3 things it can do:
+    1) Feed hungry customers
+    2) Charge customers
+    3) Provide salary to employees
+ */
+
+/*
+Thing 2) Family
+5 properties:
+    1) Size of family
+    2) Number of kids
+    3) Number of parents
+    4) Mother's name
+    5) Father's name
+3 things it can do:
+    1) Eat dinner together
+    2) Go to church
+    3) Bicker
+ */
+
+/*
+Thing 3) Synthesizer
+5 properties:
+    1) Number of keys
+    2) Number of synth voices
+    3) Name of synth
+    4) Output gain
+    5) Number of outputs
+3 things it can do:
+    1) Play sound
+    2) Change output gain amount
+    3) Change number of outputs
+ */
+
+/*
+Thing 4) Laptop
+5 properties:
+    1) Model year
+    2) Amount of RAM
+    3) Amount of storage
+    4) Screen size
+    5) Manufacturer
+3 things it can do:
+    1) Compile code
+    2) Browse the web
+    3) Play movies
+ */
+
+/*
+Thing 5) Compressor
+5 properties:
+    1) Hysteresis
+    2) Gain 
+    3) Threshold
+    4) Range
+    5) Ratio
+3 things it can do:
+    1) Limit 
+    2) Compress 
+    3) Expand 
+ */
+
+/*
+Thing 6) EQ 
+5 properties:
+    1) highBandQ
+    2) midBand1Q
+    3) midband2Q
+    4) lowbandQ
+    5) highBand gain
+3 things it can do:
+    1) low shelf shape EQ
+    2) high shelf shape EQ
+    3) peak shape EQ
+ */
+
+/*
+Thing 7) Input Section
+5 properties:
+    1) line gain
+    2) mic gain
+    3) pad amount
+    4) input noise in dB
+    5) dynamic range
+3 things it can do:
+    1) Bring signal from mic to line level
+    2) add/subtract gain of line level input
+    3) Pad a signal 30 dB
+ */
+
+/*
+Thing 8) Monitor Section
+5 properties:
+    1) selected cue output numbers
+    2) selected master output numbers
+    3) master level
+    4) headphone level
+    5) talkback level
+3 things it can do:
+    1) Route audio to different cues/playback outputs
+    2) adjust master level
+    3) adjust talkback level
+ */
+
+/*
+Thing 9) Aux Section
+5 properties:
+    1) aux send 1
+    2) aux send 2
+    3) aux send 3
+    4) aux send 1 level
+    5) aux send 2 level
```

Comment:
How does `aux send 1` differ from `aux send 1 level`?
How does `aux send 2` differ from `aux send 2 level`?
- [ ] revise

Reply:
You must first choose which aux to send to, then you select the level

Reply:
so aux send 1 could be an int and level a float